.github/workflows/cron-dagcargo-sizes.yml

```diff
@@ -23,8 +28,9 @@ jobs:
         env:
           DEBUG: '*'
           ENV: ${{ matrix.env }}
-          STAGING_PG_CONNECTION: ${{ secrets.STAGING_PG_CONNECTION }}
-          STAGING_RO_PG_CONNECTION: ${{ secrets.STAGING_PG_CONNECTION }} # no replica for staging
-          PROD_PG_CONNECTION: ${{ secrets.PROD_PG_CONNECTION }}
-          PROD_RO_PG_CONNECTION: ${{ secrets.PROD_RO_PG_CONNECTION }}
+          PROD_PG_REST_JWT: ${{ secrets.PROD_PG_REST_JWT }}
+          STAGING_PG_REST_JWT: ${{ secrets.STAGING_PG_REST_JWT }}
+          PROD_PG_REST_URL: ${{ secrets.PROD_PG_REST_URL }}
+          STAGING_PG_REST_URL: ${{ secrets.STAGING_PG_REST_URL }}
+          AFTER: ${{ inputs.after }}
```

Comment:
We moved to using a direct connection to the DB because:

1. Queries are not subject to the 30s timeout heroku dyno imposes
2. Queries do not add load to the production postgrest dyno
3. Queries can be sent to the replica database (RO - read only) where they do not add load to the production DB

Reply:
Please, if making a big change to the way cron jobs or other part of the API is working then seek concensus before doing the work.

Reply:
Thanks for the context @alanshaw. 

I wasn't aware this was the case and given the `DbClient` is exposed in the `cron` package and used in other cron jobs I didn't question if wasn't ok to be used in this one as well. 
I guess from my perspective I wasn't conscious I was introducing a big change given It was a pattern I've seen used in the package. ( Don't get me wrong, now that you pointed out is quite an obvious fundamental change indeed).

I wonder if we should add some context, and documentation to make this more obvious? 
I might add something.

That said, I agree it makes a lot more sense to use a direct connection and leverage the replica given the task is quite heavy on reads.

I'll update the PR.

Reply:
@alanshaw Is the replica pointing to a cargo replica as well?

Reply:
yes, all our usage is on a dagcargo replica (albeit the same one everywhere)

Reply:
thanks!